docker/php/config/php.ini

```diff
@@ -0,0 +1,9 @@
+memory_limit = 256M
```

Comment:
Don't think we need this file, we can just get away with the default PHP ini

Reply:
removed